doc/dev/mgmt_quickstart.md

```diff
@@ -0,0 +1,340 @@
+
+# Quickstart Tutorial - Resource Management
+
+We are excited to announce that a new set of management libraries are
+released now. Those packages share a number of new features
+such as Azure Identity support, HTTP pipeline, error-handling.,etc, and
+they also follow the new Azure SDK guidelines which create easy-to-use
+APIs that are idiomatic, compatible, and dependable.
+
+You can find the details of those new libraries
+[here](https://azure.github.io/azure-sdk/releases/latest/mgmt/dotnet.html)
+
+In this basic quickstart guide, we will walk you through how to
+authenticate to Azure using the preview libraries and start interacting
+with Azure resources. There are several possible approaches to
+authentication. This document illustrates the most common scenario.
+
+## Install the package
+
+Install the Azure Resources management core library for .NET with [NuGet](https://www.nuget.org/):
+
+```PowerShell
+Install-Package Azure.ResourceManager -Version 1.0.0
+```
+
+## Prerequisites
+Set up a way to authenticate to Azure with Azure Identity.
+
+Some options are:
+- Through the [Azure CLI Login](https://docs.microsoft.com/cli/azure/authenticate-azure-cli).
+- Via [Visual Studio](https://docs.microsoft.com/dotnet/api/overview/azure/identity-readme?view=azure-dotnet#authenticating-via-visual-studio).
+- Setting [Environment Variables](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/AuthUsingEnvironmentVariables.md).
+
+More information and different authentication approaches using Azure Identity can be found in [this document](https://docs.microsoft.com/dotnet/api/overview/azure/identity-readme?view=azure-dotnet).
+
+## Authenticate the Client
+
+The default option to create an authenticated client is to use `DefaultAzureCredential`. Since all management APIs go through the same endpoint, in order to interact with resources, only one top-level `ArmClient` has to be created.
+
+To authenticate to Azure and create an `ArmClient`, do the following:
+
+```C# Snippet:Readme_AuthClient
+using System;
```

Comment:
I think we'd better sort the using

Reply:
more like here `System` are base of every Azure libs

Reply:
just remove `System` as it's useless to this sample